app/services/reservations/confirm_reservation.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Reservations
+  class ConfirmReservation
+    def initialize(reservation)
+      @reservation = reservation
+    end
+
+    def call
+      return false unless booked?
```

Comment:
```suggestion
      return false unless reservation.booked?
```

You have reservation status declared as an enum so you can use an advantages of this approach and remove your helper [methods](https://api.rubyonrails.org/v7.0.3.1/classes/ActiveRecord/Enum.html).